ZeeHomeScreen-iOS.xcodeproj/xcuserdata/antonklysa.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please don't put this in version control. We should probably add those files to .gitignore.